timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -219,6 +245,8 @@ def fit(
         if hyperparameters is None:
             hyperparameters = "default"
 
+        train_data, tuning_data = self._check_and_prepare_data_frames(train_data, tuning_data)
```

Comment:
Nitpick: Maybe the code would be a bit cleaner if
-  `_check_and_prepare_data_frame` always took one argument (`Union[TimeSeriesDataFrame, None]`)
- we would call `_check_and_prepare_data_frame` separately for train & tuning data here

This will remove some nesting in `_check_and_prepare_data_frame` and allow us to replace `(train_data,) = ...` with `train_data = ...` in a few places